File: docs-examples/example-java/src/test/java/io/micronaut/configuration/openapi/docs/OpenApiExcludeTest.java
package io.micronaut.configuration.openapi.docs;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertFalse;

class OpenApiExcludeTest {

    @Test
    void excludedControllersAreNotDocumented() {
        var paths = OpenApiSpec.load().getPaths();

        assertFalse(paths.containsKey("/old"));
        assertFalse(paths.containsKey("/internal"));
    }
}

File: docs-examples/example-kotlin/src/test/kotlin/io/micronaut/configuration/openapi/docs/OpenApiExcludeTest.kt
package io.micronaut.configuration.openapi.docs

import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Test

class OpenApiExcludeTest {

    @Test
    fun excludedControllersAreNotDocumented() {
        val paths = OpenApiSpec.load().paths

        assertFalse(paths.containsKey("/old"))
        assertFalse(paths.containsKey("/internal"))
    }
}
